tests/test_requirements/aws_test_requirements.txt

```diff
@@ -0,0 +1,8 @@
+runhouse[aws]
+pytest
+tqdm
+datasets
+dask
```

Comment:
since there is lot of duplicate requirements between base testing reqs and all these specific testing ones, think I would be in favor of having this directory something like `/additional_requirements/`, and removing the overlaps with `tests/requirements.txt`.

this way, it's easier to read/keep track of, there's no need to propagate updates in base reqs.txt into each and every sub type, and adding new reqs files for future features is shorter too.